pkg/cmd/policy_validate.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2016-2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pulumi/pulumi/pkg/backend"
+	"github.com/pulumi/pulumi/sdk/go/common/util/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func newPolicyValidateCmd() *cobra.Command {
+	var argConfig string
+
+	var cmd = &cobra.Command{
+		Use:   "validate <org-name>/<policy-pack-name> <version>",
+		Args:  cmdutil.ExactArgs(2),
+		Short: "[PREVIEW] Validate Policy Pack configuration against config schema",
+		Long:  "Validate Policy Pack configuration against the specified version of its configuration schema.",
+		Run: cmdutil.RunFunc(func(cmd *cobra.Command, cliArgs []string) error {
+			// Obtain current PolicyPack, tied to the Pulumi service backend.
+			policyPack, err := requirePolicyPack(cliArgs[0])
+			if err != nil {
+				return err
+			}
+
+			// Get version from cmd argument
+			version := &cliArgs[1]
+
+			// Load the configuration from the user-specified JSON file into config object.
+			var config map[string]*json.RawMessage
+			if argConfig != "" {
+				config, err = loadPolicyConfigFromFile(argConfig)
+				if err != nil {
+					return err
+				}
+			}
+
+			err = policyPack.Validate(commandContext(),
+				backend.PolicyPackOperation{
+					VersionTag: version,
+					Scopes:     cancellationScopes,
+					Config:     config,
+				})
+			if err != nil {
+				return err
+			}
+			fmt.Println("Policy Pack configuration is valid.")
+			return nil
+		}),
+	}
+
+	cmd.Flags().StringVar(&argConfig, "config", "",
+		"The file path for the Policy Pack configuration file")
+	cmd.MarkFlagRequired("config") // nolint: errcheck
```

Comment:
What if you want to validate an empty config is okay? I think it may be okay to no support this.. just bc if this field isnt required then things get weird / easy to make a mistake... and you can always just pass a file with an empty `{}` to validate the empty config is ok

Reply:
I marked the flag as required, because at least the way it is written now, the command requires a config file or will error out. So I just made it required up front. You can however pass a file with "{}" if you want. Not sure if this is what you are asking though.... maybe I am misunderstanding.